src/oc/web/components/ui/section_editor.cljs

```diff
@@ -371,25 +373,33 @@
             [:button.mlb-reset.delete-bt
               {:on-click (fn []
-                          (alert-modal/show-alert
-                           {:icon "/img/ML/trash.svg"
-                            :action "delete-section"
-                            :message [:span
-                                       [:span "Are you sure?"]
-                                       (when (-> section-data :entry-count pos?)
-                                         [:span
-                                           " This will delete the section and "
-                                           [:strong "all"]
-                                           " its posts, too."])]
-                            :link-button-title "No"
-                            :link-button-cb #(alert-modal/hide-alert)
-                            :solid-button-style :red
-                            :solid-button-title "Yes, I'm sure"
-                            :solid-button-cb (fn []
-                                               (section-actions/section-delete
-                                                 (:slug section-data))
-                                               (alert-modal/hide-alert)
-                                               (dismiss))}))}
+                          (when-not last-section-standing
+                            (alert-modal/show-alert
+                             {:icon "/img/ML/trash.svg"
+                              :action "delete-section"
+                              :message [:span
+                                         [:span "Are you sure?"]
+                                         (when (-> section-data :entry-count pos?)
+                                           [:span
+                                             " This will delete the section and "
+                                             [:strong "all"]
+                                             " its posts, too."])]
+                              :link-button-title "No"
+                              :link-button-cb #(alert-modal/hide-alert)
+                              :solid-button-style :red
+                              :solid-button-title "Yes, I'm sure"
+                              :solid-button-cb (fn []
+                                                 (section-actions/section-delete
+                                                   (:slug section-data))
+                                                 (alert-modal/hide-alert)
+                                                 (dismiss))})))
+               :data-toggle "tooltip"
+               :data-placement "top"
+               :data-container "body"
+               :title (if last-section-standing
+                       "You cannot delete the last remaining section."
+                       "Delete this section and all its posts.")
+               :class (when last-section-standing "disabled")}
               "Delete section"])
```

Comment:
Should we remove the delete button completely?

Also, I can delete all the sections.
http://recordit.co/OIfp05eEN5

Reply:
We don't want to remove the delete button because we want to let the users know why they can't delete the section.. Fixing now the issue